validator/accounts/account.go

```diff
@@ -126,3 +130,32 @@ func Exists(keystorePath string) (bool, error) {
 	}
 	return true, err
 }
+
+// CreateValidatorAccount creates a validator account from the given cli context.
+func CreateValidatorAccount(path string, passphrase string) (string, string, error) {
+	if passphrase == "" {
+		reader := bufio.NewReader(os.Stdin)
+		logrus.Info("Create a new validator account for eth2")
```

Comment:
Any reason not to use the prefixed `log` here?

Reply:
Nope; fixed.